scripts/update_standlone_data

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env Rscript
+
+root_dir <- here::here()
+
+demp <- readRDS(testthat::test_path("testdata/demographic_projection_object_adult.rds"))
+parameters <- readRDS(testthat::test_path("testdata/projection_parameters_adult.rds"))
+
+frogger:::serialize_r_to_tensor(demp$basepop[, , 1], "inst/fit_model/data/basepop")
+frogger:::serialize_r_to_tensor(demp$Sx, "inst/fit_model/data/survival")
+frogger:::serialize_r_to_tensor(demp$netmigr, "inst/fit_model/data/net_migration")
+frogger:::serialize_r_to_tensor(demp$asfr, "inst/fit_model/data/age_sex_fertility_ratio")
+frogger:::serialize_r_to_tensor(demp$births_sex_prop, "inst/fit_model/data/births_sex_prop")
+frogger:::serialize_r_to_tensor(parameters$incidinput, "inst/fit_model/data/incidence_rate")
+frogger:::serialize_r_to_tensor(parameters$incrr_age, "inst/fit_model/data/incidence_rate_relative_risk_age")
+frogger:::serialize_r_to_tensor(parameters$incrr_sex, "inst/fit_model/data/incidence_rate_relative_risk_sex")
```

Comment:
Document and tidy this up. Why would someone run this script?

Reply:
Could run this on CI and see if the files have changed

Reply:
Made a ticket for this https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-4503/Add-test-which-checks-the-standalone-test-data-is-not-out-of-date